src/app/component/Page.react.js

```diff
@@ -83,10 +84,22 @@ export default class Page extends Component {
     return (yiq >= 128) ? colors.BLACK_BRIGHT : colors.GRAY_BRIGHT
   }
 
+  sortProps(propsDefinition) {
+    if (Object.keys(propsDefinition).length === 0)
+      return null
+    else
+      return (
+        Object.keys(propsDefinition)
```

Comment:
If you used a `Map`, what you would get back after sorting it would be `OrderedMap` (entries in the map get sorted by the key) so all that needs to be here is `return propsDefinition.sort()`.